.github/workflows/macos.yml

```diff
@@ -60,6 +60,16 @@ jobs:
         run: |
           ./build/macos/minetest.app/Contents/MacOS/minetest --run-unittests
 
+      - name: CPack
+        run: |
+          cd build
+          cpack -G ZIP -B macos
+
+      - name: Package Clean
+        run: |
+          rm -r build/macos/_CPack_Packages
+          rm -rf build/macos/minetest.app
```

Comment:
Also instead of cleaning I think you can just point the upload task to `./build/macos/*.zip`

Reply:
Yep, will do. I think I started with that to match the other builds (which do cleanup for some reason?) but no need.